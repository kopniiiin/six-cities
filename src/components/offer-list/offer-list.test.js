import React from "react";
import {shallow} from "enzyme";

import OfferList from "./offer-list.jsx";

import testMocks from "../../test-mocks/offer-list.js";

describe(`snapshot test: OfferList component`, () => {
  it(`should render correctly`, () => expect(
      shallow(<OfferList {...testMocks}/>)
  ).toMatchSnapshot());
});
