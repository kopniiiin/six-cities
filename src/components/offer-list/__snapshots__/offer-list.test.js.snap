// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test: OfferList component should render correctly 1`] = `
<div
  className="cities__places-list places__list tabs__content"
>
  <OfferCard
    id="4"
    isFavorite={false}
    isPremium={false}
    key="4"
    name="Paper place"
    onMouseEnter={[Function]}
    onNameClick={[Function]}
    photos={
      Array [
        Object {
          "alt": "Place photo",
          "src": "img/room.jpg",
        },
      ]
    }
    price={4}
    rating={4}
    type="room"
  />
</div>
`;
